debian/crest-prov.install

```diff
@@ -0,0 +1,3 @@
+src/metaswitch/crest/tools /usr/share/clearwater/crest-prov/src/metaswitch/crest/
+src/metaswitch/crest/tools/sstable_provisioning/* /usr/share/clearwater/crest-prov/tools/sstable_provisioning
```

Comment:
If you want to avoid a docs update, you can probable put these in /crest/, but just keep them in a different venv. Probably neater to separate them though.